clients/js/test/close/nonFungible.test.ts

```diff
@@ -23,6 +35,88 @@ const closeDestination = publicKey(
   'GxCXYtrnaU6JXeAza8Ugn4EE6QiFinpfn8t3Lo4UkBDX'
 );
 
+test.skip('it can close t22 ownerless metadata for a non-fungible with zero supply', async (t) => {
+  const umi = await createUmi();
+  const mint = generateSigner(umi);
+  const closeAuthority = createSignerFromKeypair(
+    umi,
+    umi.eddsa.createKeypairFromSecretKey(
+      new Uint8Array(
+        JSON.parse(
+          readFileSync(
+            '/Users/kelliott/Metaplex/keys/C1oseLQExhuEzeBhsVbLtseSpVgvpHDbBj3PTevBCEBh.json'
+          ).toString()
+        )
+      )
+    )
+  );
+
+  await createV1(umi, {
+    mint,
+    name: 'My NFT',
+    uri: 'https://example.com/my-nft.json',
+    tokenStandard: TokenStandard.NonFungible,
+    splTokenProgram: SPL_TOKEN_2022_PROGRAM_ID,
+    sellerFeeBasisPoints: percentAmount(0),
+  }).sendAndConfirm(umi);
+
+  // And we derive the associated token account from SPL Token 2022.
+  const token = findAssociatedTokenPda(umi, {
+    mint: mint.publicKey,
+    owner: umi.identity.publicKey,
+    tokenProgramId: SPL_TOKEN_2022_PROGRAM_ID,
+  });
+
+  // When we mint one token.
+  await mintV1(umi, {
+    mint: mint.publicKey,
+    token,
+    tokenOwner: umi.identity.publicKey,
+    amount: 1,
+    splTokenProgram: SPL_TOKEN_2022_PROGRAM_ID,
+    tokenStandard: TokenStandard.NonFungible,
+  }).sendAndConfirm(umi, { send: { skipPreflight: true } });
+
+  const asset = await fetchDigitalAsset(umi, mint.publicKey);
+
+  await burnToken22(umi, {
+    account: token,
+    mint: mint.publicKey,
+    amount: 1,
+  }).sendAndConfirm(umi);
+
+  const metadataLamports = await umi.rpc.getBalance(asset.metadata.publicKey);
+
+  if (asset.edition === undefined) {
+    t.fail('Expected edition to exist');
+  }
+
+  const masterEditionLamports = await umi.rpc.getBalance(
+    // @ts-ignore
```

Comment:
what is ts-ignore here for?  Is it because asset.edition might be undefined and it doesn't clear it with the `check asset.edition === undefined` above?  Could you use `assertAccountExists` in umi to fix that?

Reply:
Maybe it's my mint config but for some reason it refused to accept it wasn't undefined even with an explicit check.